docs/static/releasenotes.asciidoc

```diff
@@ -0,0 +1,83 @@
+* Offline plugin management: We are introducing a new mechanism to manage plugins in an offline(air-gapped) 
+  environment. The previous approach would not completely download the plugins' dependency chain on the staging
+  machine. A new command called `prepare-offline-pack` will replace the existing `pack` and `unpack` subcommands 
+  in the `logstash-plugin` CLI to manage offline plugins. The `pack` and `unpack` subcommands have been 
+  deprecated ({lsissue}6393[Issue 6393]).
+* Fixed an issue where plugins couldn't be installed when HTTP proxy was enabled in Logstash 
+  deployments ({lsissue}6044[Issue 6044], {lsissue}5777[Issue 5777], {lsissue}5966[Issue 5966]).
+* Improved user facing error when `logstash.yml` is invalid ({lsissue}6509[Issue 6509]).
+* Fixed an issue where Logstash slowlog field `took_in_millis` reported values in microseconds instead of 
+  milliseconds ({lsissue}6476[Issue 6476]).
+* Added explicit `fsync` when writing to the checkpoint file to ensure date has been flushed to disk.
+* `jvm.options` is now loaded relative to the location of `logstash.yml` or if a custom `path.settings` is 
+  specified ({lsissue}6379[Issue 6379]).
+* Added ability to create a startup init script to the `system-install` utility ({lsissue}6148[Issue 6148]).
+
+[float]
+==== Input Plugins
+
+*`Kafka`*:
+
+* Made error reporting clearer when the connection to Kafka broker fails.
+* Made a change to set `kerberos` option only when using GSSAPI.
+
+*`Ganglia`*:
+
+* Store the name and value of ganglia metrics (https://github.com/logstash-plugins/logstash-input-ganglia/issues/2[Issue 2]).
+
+*`RabbitMQ`*
+
+* Fixed issue where metadata would be broken for strings values larger than 255 chars in message headers (https://github.com/logstash-plugins/logstash-input-rabbitmq/issues/94[Issue 94])
+
+==== Filter Plugins
+
+*`GeoIP`*:
+
+* `target` field should be merged and not completely overwritten by looked up GeoIP data (https://github.com/logstash-plugins/logstash-filter-geoip/issues/98[Issue 98]).
+
+*`Sleep`*:
+
+* Fixed bug causing a crash when `every` config option is set (https://github.com/logstash-plugins/logstash-filter-sleep/issues/5[Issue 5]).
+
+==== Codec Plugins
+
+*`Netflow`*:
+
+* Added support for VMware VDS IPFIX.
+* Fixed 0-length scope field length (Netflow 9, Juniper SRX)
+  
+==== Output Plugins
+
+*`Elasticsearch`*:
+
+* Previously users could specify a proxy configuration as a hash. This has been disabled due to security reasons.
+* Proxy feature has been fixed to work when authentication is specified in the URL (https://github.com/logstash-plugins/logstash-output-elasticsearch/issues/516[Issue 516]).
+* Fixed a bug that forced users to URL encode the `password` option. Passwords with special characters can be used directly 
+  in the URL or in the `password` option.
+
+*`S3`*:
+
+* This plugin has been updated to use the v2.0 of the AWS SDK which brings in many updates and bug fixes.
```

Comment:
Just to maintain consistency (for easy scanning), should say, "Updated this plugin to use the v2.0 of the AWS SDK...."